server/user/model.ts

```diff
@@ -186,7 +194,7 @@ export class User extends ModelWithPassport<
 	googleScholar!: string | null;
 
 	@Column(DataType.DATE)
-	resetHashExpiration!: Date | null;
```

Comment:
`resetHasExpiration` is always treated as `number`, so I made it `number` instead of `string` like with the others.